docs/index.rst

```diff
@@ -5,6 +5,6 @@
    :hidden:
 
    getting_started/intro.rst
-   user_guide/intro.rst
-   api.rst
+   user_guide/index.rst
+   api/index.rst
```

Comment:
I swithced to API only in the tabs in core to make it shorter. WHat do you think?


```suggestion
   API <api/index.rst>
```